CL/cl_ext.h

```diff
@@ -578,6 +578,49 @@ typedef cl_uint  cl_queue_throttle_khr;
 #define CL_DEVICE_MAX_NAMED_BARRIER_COUNT_KHR       0x2035
 
 
+/*********************************
+* cl_khr_extended_versioning
+*********************************/
+
+#define CL_VERSION_MAJOR_BITS_KHR (10)
+#define CL_VERSION_MINOR_BITS_KHR (10)
+#define CL_VERSION_PATCH_BITS_KHR (12)
+
+#define CL_VERSION_MAJOR_MASK_KHR ((1 << CL_VERSION_MAJOR_BITS_KHR) - 1)
+#define CL_VERSION_MINOR_MASK_KHR ((1 << CL_VERSION_MINOR_BITS_KHR) - 1)
+#define CL_VERSION_PATCH_MASK_KHR ((1 << CL_VERSION_PATCH_BITS_KHR) - 1)
+
+#define CL_VERSION_MAJOR_KHR(version) ((version) >> (CL_VERSION_MAJOR_BITS_KHR + CL_VERSION_PATCH_BITS_KHR))
```

Comment:
This currently uses `(CL_VERSION_MAJOR_BITS_KHR + CL_VERSION_PATCH_BITS_KHR)`, but I think it should be `(CL_VERSION_MINOR_BITS_KHR + CL_VERSION_PATCH_BITS_KHR)` instead?  Note MINOR vs. MAJOR.

Both `MAJOR_BITS` and `MINOR_BITS` are currently `10`, so functionally this is fine right now, but it's confusing and it could be wrong if we ever change the number of major or minor bits in the future or copy-paste this macro for some other usage.

Reply:
Thanks for the review. Good catch! Just double-checked the definitions in the spec: they're fine.